frontend/src/components/WaitingRoom/VideoLoading/VideoLoading.tsx

```diff
@@ -11,7 +11,7 @@ const VideoLoading = (): ReactElement => {
   return (
     <div
       data-testid="VideoLoading"
-      className="absolute  flex items-center justify-center w-[584px] h-[328px] rounded-2xl bg-black"
+      className="absolute  flex items-center justify-center w-dvw h-[328px] rounded-2xl bg-black"
```

Comment:
nit while you're in the area - would you mind removing the extra space between `absolute` and `flex`? otherwise LGTM 🙏